src/main/scala/cromwell/Main.scala

```diff
@@ -333,18 +341,25 @@ class Main private[cromwell](enableTermination: Boolean, managerSystem: () => Wo
     a <- Actions.values find (_.toString == argCapitalized)
   } yield a
 
-  private[this] def loadWdl(path: String)(f: WdlNamespace => Int): Int = {
-    Try(WdlNamespace.load(new JFile(path), WorkflowManagerActor.BackendType)) match {
-      case Success(namespace) => f(namespace)
-      case Failure(t) =>
-        println(t.getMessage)
-        exit(1)
+  private[this] def loadWdl(args: Seq[String])(f: WdlNamespace => Int): Int = {
+    /*
+      As of this writing all functions which call this have >= 2 args and are such that args(0) is the path
+      and args(1) is the backend type. Those functions will be going away soon so it's unlikely that something
+      runs afoul of this 
```

Comment:
How hard would it be to not break the command line?  I get you just wanted quick and dirty, but now the docs are broken too, and the help docs are wrong, and it'd be very messy to release Cromwell 0.15 like this.  

Can we just default this to `BackendType.LOCAL` or something?